cli/wallet/nep5.go

```diff
@@ -41,7 +41,7 @@ func newNEP5Commands() []cli.Command {
 		walletPathFlag,
 		tokenFlag,
 		cli.StringFlag{
-			Name:  "addr",
+			Name:  "address, a",
```

Comment:
Usage text should also be fixed (line 81).

Reply:
Fixed.